### ISSUE TITLE
Combined courses not saved properly

### ISSUE THREAD

**Issue by Maxcdr** [Courseplay/courseplay#1101]
Searched in other submited issues but found nothing.

Map: Coldborough Farm 2015 / Standard vehicles
CP v4.01.0090

This issue often happens but not everytime.
I have a vehicle with 8 combined courses like this for example:

![fsscreen_2015_03_10_16_07_52](https://cloud.githubusercontent.com/assets/10348839/6577737/4edcbf2c-c740-11e4-9ea3-f9a4b085e568.png)

I save my game and close FS15. When I reopen FS15 and load my game, vehicle has 7 combined courses which look like this:

![fsscreen_2015_03_10_16_06_01](https://cloud.githubusercontent.com/assets/10348839/6577778/8e451d62-c740-11e4-88b3-0cd8bcb7d427.png)

So each time I have to clear current course and  reload each 8 courses.

**Comment by gui7545** [comment]
Because you are using temporary course. I'am sure that you don't saved the field course, right ?
That is why this course is missing

**Comment by Maxcdr** [comment]
All used courses are saved including field courses...

**Comment by sootysax** [comment]
this has been happening to me since the start of Courseplay.   i always save my combined courses as a new course before needing to exit/reenter a game.

**Comment by Karlshoj** [comment]
I agree with sootysax

**Comment by Maxcdr** [comment]
sootysax you're my hero ! I just discovered this feature thanks to you !

**Comment by gui7545** [comment]
Sorry for this question: What do you do to display the courseplay courses on the map ?

**Comment by Maxcdr** [comment]
@gui7545 : not sure to understand your question... Could you rephrase?

**Comment by gui7545** [comment]
On the right side of your screenshot, there is a map with courseplay courses display on it. It is a feature of a mod ? map ? or courseplay ?

**Comment by Maxcdr** [comment]
Click on the eye at top left corner of CP HUD.

**Comment by sootysax** [comment]
click the icon shaped like an eye.

**Comment by gui7545** [comment]
Awesome, thanks, I think we can close the issue ? @Maxcdr ?

**Comment by AndyK70** [comment]
@gui7545 
> Because you are using temporary course. I'am sure that you don't saved the field course, right ?
That is why this course is missing

I did get this issue, too.
It's even when you saved the course. 
When you generate a fieldcourse, save it and then discard the fieldcourse and reload it it won't happen.

It's because in memory it is still a temporary course, even when it is saved. Only if you load it again it will become the full saved course.
Don't know, if this is a issue in coursplay, or if courseplay could get around it, by discarding and reloading it automatically right after saving a course?
